connector/subgraph/src/ConvictionVoting.ts

```diff
@@ -84,6 +88,15 @@ function _onNewStake(
     return 
   }
 
+  const config = getConfigEntity(appAddress)
+
+  if (proposal.totalTokensStaked < totalTokensStaked){
+    config.totalStaked = config.totalStaked.plus(amount)
+  } else {
+    config.totalStaked = config.totalStaked.minus(amount)
+  }
+  config.save()
```

Comment:
What does this change mean? I think we can leave a comment. 😄

Reply:
you mean the if? that is to identify if is a withdraw or a stake,  https://github.com/1Hive/conviction-voting-app/blob/a5652436af1ed56321afbc836720a0fb09abedbd/connector/subgraph/src/ConvictionVoting.ts#L33-L57